sympy/printing/codeprinter.py

```diff
@@ -166,16 +166,17 @@ def doprint(self, expr, assign_to=None):
 
     def _doprint_loops(self, expr, assign_to=None):
         # Here we print an expression that contains Indexed objects, they
-        # correspond to arrays in the generated code.  The low-level implementation
-        # involves looping over array elements and possibly storing results in temporary
-        # variables or accumulate it in the assign_to object.
+        # correspond to arrays in the generated code.  The low-level
+        # implementation involves looping over array elements and possibly
+        # storing results in temporary variables or accumulate it in the
+        # assign_to object.
 
         if self._settings.get('contract', True):
```

Comment:
I think the `contract` setting is obsoleted by `EinsteinSum` right? If it's an `EinsteinSum` that is to be processed it implies `contract=True` and if not it implies `contract=False` - right?